MainModule/Client/UI/Default/Inspect.lua

```diff
@@ -122,12 +112,11 @@ return function(data)
 
 		addGeneralEntry("Username:", player.Name, "The player's Roblox username")
 		addGeneralEntry("User ID:", player.UserId, "The player's unique Roblox user ID")
-		addGeneralEntry("Account Age:", player.AccountAge .. " ("..string.format("%.2f", player.AccountAge/365).." years)", "How long (in days) the player has been registered on Roblox")
-		addGeneralEntry("Membership:", msTypeToStr(player.MembershipType), "The player's Roblox membership type")
+		addGeneralEntry("Account Age:", player.AccountAge .. " days ("..string.format("%.2f", player.AccountAge/365).." years)", "How long the player has been registered on Roblox")
+		addGeneralEntry("Membership:", tostring(player.MembershipType):sub(21), "The player's Roblox membership type")
```

Comment:
You can use the `Name` property on Enum properties, so you should be able to simplify it only to `player.MembershipType.Name` instead.

Reply:
Alright, thanks for the info.